aviary/subsystems/propulsion/gearbox/test/test_gearbox.py

```diff
@@ -53,19 +53,26 @@ def test_gearbox_mission(self):
 
         prob.setup(force_alloc_complex=True)
 
-        prob.set_val(av.Aircraft.Engine.RPM_DESIGN, [5000, 6195, 6195], units='rpm')
-        prob.set_val(av.Dynamic.Mission.SHAFT_POWER, [100, 200, 375], units='hp')
-        prob.set_val(av.Dynamic.Mission.SHAFT_POWER_MAX, [375, 300, 375], units='hp')
+        prob.set_val(av.Dynamic.Mission.RPM + '_in', [5000, 6195, 6195], units='rpm')
```

Comment:
For me, the _in and _out variables aren't helping me remember what is what. A graphical representation would help me with this.

Reply:
I'll do that when documentation finally gets added

Reply:
see https://github.com/OpenMDAO/Aviary/pull/514#discussion_r1823530694

Reply:
Those descriptions in #514 make sense to me.